include/tvm/relay/attrs/transform.h

```diff
@@ -122,6 +122,17 @@ struct SplitAttrs : public tvm::AttrsNode<SplitAttrs> {
   }
 };
 
+struct SliceLikeAttrs : public tvm::AttrsNode<SliceLikeAttrs> {
+  Array<Integer> axes;
+
+  TVM_DECLARE_ATTRS(SliceLikeAttrs, "relay.attrs.SliceLikeAttrs") {
+    TVM_ATTR_FIELD(axes).set_default(Array<Integer>())
+        .describe("List of axes on which input data will be sliced according to the "
+                  "corresponding size of the second input. By default will slice "
+                  "on all axes. Negative axes are supported.");
```

Comment:
Negative axes are supported -> Negative axes mean counting in reverse.
Only slightly longer, and convey the exact meaning